index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/774f0ae2db.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css-reset.css">
+  <link rel="stylesheet" href="style.css">
+  <title>Apple</title>
+</head>
+<body>
+  <div class="app-container">
+    <main class="main-section">
+      <div class="nav-wrapper">
+        <nav class="global-navigation">
+          <ul class="nav-items">
+            <li><a href=""><i class="fab fa-apple"></i></a></li>
+            <li><a href="">Store</a></li>
+            <li><a href="">Mac</a></li>
+            <li><a href="">ePod</a></li>
+            <li><a href="">ePhone</a></li>
+            <li><a href="">ePad</a></li>
+            <li><a href="">eTunes</a></li>
+            <li><a href="">Support</a></li>
+          </ul>
+          <form class="nav-search" action="#">
+            <input type="search"><i class="fas fa-search"></i>
+          </form>      
+        </nav>
+      </div>
+      <div class="verse-text">
+        <a href=""><img src="./images/your_verse_title_large.png" alt=""></a>
+        <a href=""><img src="./images/your_verse_subhead_small.png" alt=""></a>
+        <a href=""><img src="./images/cta_your_verse_27pt.png" alt=""></a>
+      </div>
+    </main>
+    <aside class="iphone5s">
+      <a href=""><img src="./images/promo_iphone5s.jpg" alt=""></a>
+    </aside>
+    <aside class="iphone5c">
+      <a href=""><img src="./images/promo_30_years.jpg" alt=""></a>
+    </aside>
+    <aside class="mac">
+      <a href=""><img src="./images/promo_iphone5c.jpg" alt=""></a>
+    </aside>
+    <aside class="supplier">
+      <a href=""><img src="./images/promo_supplier_responsibility.jpg" alt=""></a>
```

Comment:
- [ ] Required - Also as this project is background-and-gradients based kindly make these images as backgrounds instead.